### PR TITLE
fixes: #1503 Linchpin clean installation with Python 3 requires pymongo

### LINKED ISSUE #1503: Linchpin clean installation with Python 3 requires pymongo
**Describe the bug**
`linchpin up` fails on missing pymongo module:
```
Traceback (most recent call last):
  File "/opt/linchpin3/.venv/bin/linchpin", line 11, in <module>
    load_entry_point('linchpin', 'console_scripts', 'linchpin')()
  File "/opt/linchpin3/.venv/lib64/python3.7/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/opt/linchpin3/.venv/lib64/python3.7/site-packages/pkg_resources/__init__.py", line 2793, in load_entry_point
    return ep.load()
  File "/opt/linchpin3/.venv/lib64/python3.7/site-packages/pkg_resources/__init__.py", line 2411, in load
    return self.resolve()
  File "/opt/linchpin3/.venv/lib64/python3.7/site-packages/pkg_resources/__init__.py", line 2417, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/opt/linchpin3/linchpin/__init__.py", line 23, in <module>
    from linchpin.rundb.drivers import DB_DRIVERS
  File "/opt/linchpin3/linchpin/rundb/drivers.py", line 4, in <module>
    from .mongodb import MongoDB
  File "/opt/linchpin3/linchpin/rundb/mongodb.py", line 10, in <module>
    from pymongo import MongoClient, DESCENDING
ModuleNotFoundError: No module named 'pymongo'
```

**To Reproduce**
1. Create python 3 virtual env `python3 -m venv .venv`
2. Install linchpin from repo: `pip install -e .`
3. Run linchpin up

**Expected behavior**
If mongo driver is not used then it shouldn't be imported

### PR DESCRIPTION
**Pull request by samvarankashyap**
Fixes #1503

### CODE REVIEW

File: linchpin/rundb/drivers.py [hunk 1]
```diff
@@ -1,13 +1,24 @@
 from __future__ import absolute_import
 
 from .tinydb import TinyRunDB
-from .mongodb import MongoDB
 
+try:
+    from .mongodb import MongoDB
+except Exception as e:
+    print(str(e))
+    print("pymongo not installed")
```
Comment (14rcole) [inline linchpin/rundb/drivers.py]: Can you change this error message to note that pymongo is not available in python 2?  Also, can you provide some indication that this is a warning?

Comment (samvarankashyap) [reply linchpin/rundb/drivers.py]: Sure @14rcole I will update it

**Comment by samvarankashyap** [comment]
@14rcole Done

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**